content/cucumber/api.md

```diff
@@ -300,7 +300,13 @@ if (scenario.isFailed()) {
 ```
 
 ```javascript
-TODO: See Ruby or Java example
+if (scenario.isFailed()) {
```

Comment:
Should be ```scenario.result.status === Status.FAILED``` https://github.com/cucumber/cucumber-js/blob/master/docs/support_files/attachments.md

Reply:
@NguyenAndrew , are you sure about this? Since I saw http://rmpestano.github.io/cukedoctor/cucumber-js/cucumber-js-documentation.html#_check_scenario_statuses. What you are recommending is for each test case inside a scenario.

Reply:
I'd go with Cucumber-js documentation over CukeDoctor documentation @TomerPacific

Reply:
In Cucumber.js (v5.1.0), when I did ```scenario.isFailed();``` the following error happened ```TypeError: scenario.isFailed is not a function```, whereas ```scenario.result.status === Status.FAILED``` works successfully.